src/Track.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace Klaviyo;
+
+use Klaviyo\Model\EventModel;
+use Klaviyo\Model\ProfileModel;
+
+class Track extends KlaviyoAPI
+{
+    /**
+     * Track Class constants
+     */
+    const TRACK = 'track';
+    const IDENTIFY = 'identify';
+
+    /**
+     * Track constructor.
+     * @param $public_key
+     * @param $private_key
+     * @param string $host
+     */
+    public function __construct($public_key, $private_key, $host = self::BASE_URL) {
+        parent::__construct($public_key, $private_key, $host = self::BASE_URL);
+
+        if ( !isset( $this->public_key ) ) {
+            throw new KlaviyoException('Public key is not defined.');
+        }
```

Comment:
You should be fine to remove this since we require both private and public keys when instantiating the klaviyo client